aspnetcore/security/authorization/secure-data.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Create an app with user data protected by authorization
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 10/14/2016
+ms.topic: article
+ms.assetid: abeb2f8e-dfbf-4398-a04c-338a613a65bc
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: security/authorization/secure-data
+---
+
+# Create an app with user data protected by authorization
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Joe Audette](https://twitter.com/joeaudette)
+
+This tutorial shows how to create a web app with user data protected by authorization. It  displays a list of contacts that Authenticated (registered) users have created. In the following image, user `rick@example.com` is signed in. User Rick can only view approved contacts and edit/delete his contacts. Only the last record, created by `rick@example.com`, displays edit and delete links
+
+![image described above](secure-data/_static/rick.png)
+
+Authenticated users can read all approved contacts but can only edit/delete their own contacts. 
+
+In the following image, `manager@contoso.com` is signed in and in the managers role. 
+
+![image described above](secure-data/_static/manager1.png)
+
+The manager can:
+* View all contacts. 
+* Edit and delete contacts she creates. 
+* Set a contact to `Approved` or `Rejected`.
+
+The following image shows the details view of a contact.
+
+![image described above](secure-data/_static/manager.png)
+
+In the following image, `admin@contoso.com` is signed in and in the administrator’s role. 
+
+![image described above](secure-data/_static/admin.png)
+
+The administrator has all privileges. She can read/edit/delete any contact and change the status of contacts.
+
+The app was created by [scaffolding](xref:tutorials/first-mvc-app-xplat/adding-model#scaffold-the-moviecontroller)  the following `Contact` model:
+
+[!code-csharp[Main](secure-data/samples/starter/Models/Contact.cs?name=snippet1)]
+
+The contact information properties (Address, Name, etc.) are displayed in the images above. `ContactId` is the primary key for the table.
+
+A `ContactIsOwnerAuthorizationHandler` authorization handler ensures that a user can only edit their data. A `ContactManagerAuthorizationHandler` authorization handler allows managers to approve or reject contacts.  A `ContactAdministratorsAuthorizationHandler` authorization handler allows administrators to approve or reject contacts and to edit/delete contacts. 
```

Comment:
@joeaudette  should I rename it `ContactUserAuthorizationHandler` ? I sort of like the original as it denotes _IsOwner_. They all start with Contact.